source/getting-started/supported-databases.txt

```diff
@@ -59,6 +59,8 @@ Relational Migrator supports the following source databases:
      - 15 and higher (Sybase IQ not supported)
      - Self hosted
 
+.. [1] `TimescaleDB hypertables <https://docs.timescale.com/use-timescale/latest/hypertables/>`__ are supported. 
```

Comment:
Even though the footnote is marked for Postgres, It might help to clarify:

"TimescaleDB hypertables are supported for PostgreSQL."

I think this will help readers who are scanning the page.